lib/sedna/algorithms/aggregation/__init__.py

```diff
@@ -12,4 +12,59 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
+"""Aggregation algorithms"""
+
+# TODO: merge with aggregation
+
+# import abc
+# from copy import deepcopy
+
+# import numpy as np
+
+# from sedna.common.class_factory import ClassFactory, ClassType
+
+# __all__ = ('FedAvg',)
 from .aggregation import *
+
+# class BaseAggregation(metaclass=abc.ABCMeta):
+#     def __init__(self):
+#         self.total_size = 0
+#         self.weights = None
+
+#     @abc.abstractmethod
+#     def aggregate(self, weights, size=0):
+#         """
+#         Aggregation
+#         :param weights: deep learning weight
+#         :param size: numbers of sample in each loop
+#         """
+
+
+# # os.environ['config_file'] = '/home/plato/examples/configs/server.yml'
+# # from plato.servers import fedavg
+
+# @ClassFactory.register(ClassType.FL_AGG)
+# class FedAvg(BaseAggregation, abc.ABC):
```

Comment:
I agree with you that move the `FedAvg`  to another file, but it will be better to delete the useless code.

Reply:
updated